tools/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/cfergeau/win32-background-launcher
```

Comment:
This should be `github.com/crc-org/win32-background-launcher/tools` (and I've seen now that cfergeau -> crc-org is addressed in one of the next commits)